deps/pjsip/third_party/zsrtp/zrtp/srtp/crypto/openssl/hmac.cpp

```diff
@@ -80,7 +80,6 @@ void* initializeSha1HmacContext(void* ctx, uint8_t* key, int32_t keyLength)
 {
     HMAC_CTX *pctx = (HMAC_CTX*)ctx;
 
-    HMAC_CTX_init(pctx);
```

Comment:
This function (initializeSha1HmacContext) seems to bed used to initialize (i.e. reset and start over) an existing context. If HMAC_CTX_init() is removed I do not think it will work properly anymore. (in case it's compiled with openssl-1.0)

Reply:
Ah yes; and actually, it was replaced with HMAC_CTX_reset in 1.1 so probably won't work properly when compiled with that too. 
Have restored this line wrapped in a `#if` and added a call to the new `HMAC_CTX_reset` method for 1.1